### PR TITLE
two trivial fixes after advanced audit upgrade to beta

### PR DESCRIPTION
**Pull request by CaoShuFeng**
This change addresses two comments from @crassirostris

https://github.com/kubernetes/kubernetes/pull/49115#discussion_r133417334
https://github.com/kubernetes/kubernetes/pull/49115#discussion_r133416208

**Release note**:
```
NONE
```

### CODE REVIEW

**Comment by crassirostris** [comment]
@CaoShuFeng Maybe address comments about tests also? I mean moving log backend tests from the generic audit package

**Comment by CaoShuFeng** [comment]
> @CaoShuFeng Maybe address comments about tests also? I mean moving log backend tests from the generic audit package

Yes, I am working on it.
About 60% done now.

**Comment by crassirostris** [comment]
@CaoShuFeng Awesome, thanks!

File: staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go [hunk 1]
```diff
@@ -88,8 +89,7 @@ var (
 	//
 	// Can we make these passable to NewGenericWebhook?
 	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
-	// TODO(audit): figure out a general way to let the client choose their preferred version
```
Comment (deads2k) [inline staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go]: Why doesn't this still stand?

Comment (crassirostris) [inline staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go]: It's just moved to a different place: https://github.com/kubernetes/kubernetes/pull/50824/files#diff-6b8b85c8f6acaf59bdf51e27a6b605bbR74

Comment (CaoShuFeng) [inline staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go]: Some suggestions about this TODO:
https://github.com/kubernetes/kubernetes/issues/48561#issuecomment-319225619

**Comment by CaoShuFeng** [comment]
This two TODO has be done be soltysh

**Pull request closed without merging**